src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -103,6 +106,41 @@ def __init__(
                 tupleize_cols=tupleize_cols,
             )
 
+    def to_pandas_warning(self, func_name: str) -> None:
+        """
+        Helper method to notify users if they are using a method that currently calls to_pandas()
+        """
+        WarningMessage.single_warning(
+            f"The method {func_name} currently calls to_pandas() and materializes data. In future updates, this method will be lazily evaluated"
+        )
```

Comment:
Can we move this method be part of the WarningMessage class in warning_message.py instead of here?